.github/workflows/wheel-builder.yml

```diff
@@ -286,8 +285,10 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
       - name: Configure OpenSSL
         run: |
-            echo "INCLUDE=C:/openssl-${{ matrix.WINDOWS.WINDOWS }}/include;$INCLUDE" >> $GITHUB_ENV
-            echo "LIB=C:/openssl-${{ matrix.WINDOWS.WINDOWS }}/lib;$LIB" >> $GITHUB_ENV
+            echo "OPENSSL_DIR=C:/openssl-${{ matrix.WINDOWS.WINDOWS }}" >> $GITHUB_ENV
+            echo "OPENSSL_STATIC=1" >> $GITHUB_ENV
```

Comment:
Should we be passing this in the windows CI builder as well? I think we only copy over the static libraries on Windows (https://github.com/pyca/infra/blob/main/windows/openssl/build_openssl_win64.bat#L11-L12), so maybe it works without even passing this?

Reply:
Yeah, if only static is available it'll default to it.

But either way, it doesn't _really_ matter in CI, it's only critical for wheels.